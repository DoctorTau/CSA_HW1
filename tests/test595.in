320
5848 3786 -4774 -7008 683 -591 -7425 -7582 7055 -5292 -2298 6932 9558 4200 -182 9422 -9424 4638 -1769 -7173 -1571 -6071 7293 4864 2786 -1276 -5172 1074 7918 4442 -4751 1535 4101 7325 3689 -8680 4161 -7473 -6968 -6971 -3959 -8001 -8787 1020 320 5832 7600 -2082 -1242 3949 -1912 -5466 -4608 -449 -8943 5913 -9717 -9679 -8767 3643 9471 7029 7115 6693 -5277 -9425 -3811 8554 4938 -1741 -6551 9139 -6670 -1358 2578 -363 9329 -8893 -5422 -5205 1359 -6391 7934 4748 -2777 2333 3626 -5158 8435 7371 -538 -6863 8710 -1958 -1681 2651 -325 6985 173 8419 695 482 8424 3948 -9841 3836 4269 9214 -4417 -4912 7015 1286 -6984 7305 -481 994 2590 -5580 9929 3340 847 5640 6263 -5434 2792 -3484 8556 -5852 9515 -4836 9856 459 6263 -3188 9015 7216 -3217 4848 -2476 -3316 -2766 -9427 837 464 8481 596 -5224 2437 489 9452 -2038 -2769 -2702 9410 -2668 8371 2340 -5131 -3975 8787 7377 407 -1144 -2059 -1573 9460 4470 -14 4611 -5311 -4366 -554 -1445 -7917 9112 5376 9624 -4199 -7285 -4322 4581 -3672 -8287 2492 1981 -2417 7434 5839 7066 -9081 7895 -5298 -5288 7112 -1182 -7610 393 5348 7795 -1005 9046 7239 1558 2862 -4239 1229 9583 8709 4335 -767 387 891 5442 -278 2019 4572 -4534 -5300 6189 -8185 -2254 3176 -9889 9241 694 -6478 653 993 1209 -6226 5937 -660 -1142 -2586 -160 -2612 9058 -8085 3546 -9786 1594 -3729 5649 -925 6370 -863 9026 751 -7067 3180 -9213 -1037 -3358 366 -3451 -9505 2043 1059 -1067 7124 8312 1960 728 -4168 -1762 -8292 -8556 -9832 2435 -88 6987 5407 5969 -3367 -1352 -3677 3551 5069 4072 1292 6466 6370 -4574 5952 -7915 2750 -5065 -7647 1584 -2029 -8675 2213 -1472 -7704 -1200 -8073 1418 2124 -85 -3185 -6246 -8208 -2930 697 4364 -7628 -7700 3723 -1927 4176 6884 8489 -8685 -7804 3158 869 696 6452 8310 -3019 