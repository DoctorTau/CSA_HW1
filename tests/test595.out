-4774 -7008 -7582 -5292 -2298 -182 -9424 -1276 -5172 -8680 -6968 -2082 -1242 -1912 -5466 -4608 -6670 -1358 -5422 -5158 -538 -1958 -4912 -6984 -5580 -5434 -3484 -5852 -4836 -3188 -2476 -3316 -2766 -5224 -2038 -2702 -2668 -1144 -14 -4366 -554 -4322 -3672 -5298 -5288 -1182 -7610 -278 -4534 -5300 -2254 -6478 -6226 -660 -1142 -2586 -160 -2612 -9786 -3358 -4168 -1762 -8292 -8556 -9832 -88 -1352 -4574 -1472 -7704 -1200 -6246 -8208 -2930 -7628 -7700 -7804 